pkg/manifests/manifests.go

```diff
@@ -1496,6 +1523,40 @@ func (f *Factory) setupQueryLogFile(p *monv1.Prometheus, queryLogFile string) er
 	return nil
 }
 
+// setupProfilesToIgnore configures the label selectors of the Prometheus ("p")
+// to select any ServiceMonitor's or PodMonitor's that doesn't have the scrape
+// profile label or that matches the Scrape Profile ("sp").
+func (f *Factory) setupProfilesToIgnore(p *monv1.Prometheus, sp ScrapeProfile) error {
```

Comment:
```suggestion
func setupProfilesToIgnore(p *monv1.Prometheus, sp ScrapeProfile) error {
```
Iiuc `setupProfilesToIgnore` doesn't actually use the receiver argument?